Makefile

```diff
@@ -103,37 +105,31 @@ diff_pylint_report: pylint_report.txt
 	diff-quality --violations=pylint pylint_report.txt
 
 .coverage: $(PYSOURCES)
-	coverage run --branch --source=${MODULE} setup.py test
-	coverage run --append --branch --source=${MODULE} \
-		-m schema_salad.main \
+	rm -f .coverage
+	$(COVBASE) setup.py test
+	$(COVBASE) -m schema_salad.main \
 		--print-jsonld-context schema_salad/metaschema/metaschema.yml \
 		> /dev/null
-	coverage run --append --branch --source=${MODULE} \
-		-m schema_salad.main \
+	$(COVBASE) -m schema_salad.main \
```

Comment:
@mr-c can you tell what is happening here? I know that this command invokes ``schema_salad`` functions which increases the coverage but exactly is in ``metaschema``?